### PR TITLE
Fix submission of other answers

### PR DESCRIPTION
**Pull request by Chartman123**
Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

**Comment by Chartman123** [comment]
@AIlkiv Perhaps you could also have another look at this. Current main is broken if you allow custom answers and also the validation on the server side. Statistics are also not counted correctly.

@susnux This is my first attempt to fix the problem that I told you about.

I'm currently not really sure where this is coming from, might be my change that fixed the output for XML in #1705

File: src/components/Questions/QuestionMultiple.vue [hunk 1]
```diff
@@ -62,7 +62,7 @@
 					{{ answer.text }}
 				</NcCheckboxRadioSwitch>
 				<div v-if="allowOtherAnswer" class="question__other-answer">
-					<NcCheckboxRadioSwitch :checked.sync="questionValues"
+					<NcCheckboxRadioSwitch :checked.sync="values"
```
Comment (susnux) [inline src/components/Questions/QuestionMultiple.vue:65]: This is not valid, you shall not modify properties (`values` is a property so you should instead only set it like `:checked="values"` but handle the update with a method like `@update:values="onValuesChange"`

**Comment by susnux** [comment]
> I'm currently not really sure where this is coming from, might be my change that fixed the output for XML in #1705

Yes thats at least the reason for the backend issues, for the front end the code was not that error resilient.

**Comment by AIlkiv** [comment]
@Chartman123 Is the problem still there? Can you provide examples of how to reproduce?

**Comment by Chartman123** [comment]
@AIlkiv we could already fix it :) it was introduced with my change of the backend to fix XML output

**Pull request closed without merging**